### PR TITLE
Add filter

### PR DESCRIPTION
**Pull request by academey**

### CODE REVIEW

File: app/helpers/formatPapersQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+export function formatPapersQuery({ text, yearFrom, yearTo }: IFormatPapersQueryParams) {
+  const resultQuery = `text=${text},year=${yearFrom || ""}:${yearTo || ""}`;
+
+  return encodeURIComponent(resultQuery);
+}
+
+export function objectifyPapersQuery(query: string): { text: string; yearFrom?: number; yearTo?: number } {
+  const textIndex = query.search("text=");
+  const yearFromIndex = query.search("year=");
+  const yearToIndex = query.search(":");
```
Comment (TylorShin) [inline app/helpers/formatPapersQuery.tsx]: How to get this value when there is additional field with colon?

File: app/helpers/formatPapersQuery.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+interface IFormatPapersQueryParams {
```
Comment (TylorShin) [inline app/helpers/formatPapersQuery.tsx]: I strongly suggest to make class has methods for both parser and builder.

File: app/root.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700");
```
Comment (TylorShin) [inline app/root.scss]: 🧐

Comment (academey) [reply app/root.scss]: Designer wants it

File: app/helpers/formatPapersQuery.tsx (hunk 1 above)
Comment (academey) [reply app/helpers/formatPapersQuery.tsx]: I will fix it.

File: app/helpers/formatPapersQuery.tsx (hunk 2 above)
Comment (academey) [reply app/helpers/formatPapersQuery.tsx]: I don't think so. I think to make functions a class is more difficult to manage code.

**Pull request merged**